app/layout/aside_feed.phtml

```diff
@@ -52,7 +52,7 @@
 		?>
 		<li id="tags" class="tree-folder category tags<?= $t_active ? ' active' : '' ?>" data-unread="<?= format_number($this->nbUnreadTags) ?>">
 			<div class="tree-folder-title">
-				<a class="dropdown-toggle" href="#"><?= _i($t_active ? 'up' : 'down') ?></a>
+				<a class="dropdown-toggle" href="#"><?= _i($t_active || (strlen($class) < 1) || (format_number($this->nbUnreadTags) > 0) ? 'up' : 'down') ?></a>
```

Comment:
```suggestion
				<a class="dropdown-toggle" href="#"><?= _i($t_show ? 'up' : 'down') ?></a>
```

Reply:
Solution could be sooooo simple :) Thanks.
I haven't seen it.

Reply:
P.S. You can just commit a suggestion from the Web UI here, no need to make a manual edit and commit

Reply:
I know, but I had to test it locally first ;) So I had to takeover it first into my source code before commiting.